### ISSUE TITLE
Installation Problem

### ISSUE THREAD

**Issue by yzy1121** [ICICLE-ai/Readme-guide#1]
Is portaudio19-dev needed for the installation? I need to install it first manually to start the backend server.

**Comment by RumaliPerera** [comment]
Yes I think so. This is from the speech to text module. I think you can ask Eric for further clarifications on this.
